content/manual-references.json

```diff
@@ -18,5 +18,117 @@
       "family": "Himmelstein"
     }
   ]
+},
+{
+  "type": "thesis",
+  "standard_citation": "raw:inference",
+  "URL": "https://upcommons.upc.edu/handle/2117/95691",
+  "publisher": "Universitat Politècnica de Catalunya",
+  "title": "Link prediction in large directed graphs",
+  "issued": {
+    "date-parts": [
+      [
+        2015
+      ]
+    ]
+  },
+  "author": [
+    {
+      "given": "Dario",
+      "family": "Garcia Gasulla"
+    }
+  ]
+},
+{
+  "type": "article",
+  "standard_citation": "raw:rwr",
+  "URL": "http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.85.8179&rep=rep1&type=pdf",
+  "publisher": "10th ACM SIGKDD Conference 2004",
+  "title": "Automatic Multimedia Cross-modal Correlation Discovery",
+  "issued": {
+    "date-parts": [
+      [
+        2004
+      ]
+    ]
+  },
+  "author": [
+    {
+      "given": "Jia-Yu",
+      "family": "Pan"
+    },
+    {
+      "given": "Hyung-Jeong",
+      "family": "Yang"
+    },
+    {
+      "given": "Christos",
+      "family": "Faloutsos"
+    },
+    {
+      "given": "Pinar",
+      "family": "Duygulu"
+    }
+  ]
+},
+{
+  "type": "article",
+  "standard_citation": "raw:laplacian",
+  "URL": "https://papers.nips.cc/paper/2506-learning-with-local-and-global-consistency.pdf",
+  "publisher": "NIPS 2003",
+  "title": "Learning with local and global consistency",
+  "issued": {
+    "date-parts": [
+      [
+        2003
+      ]
+    ]
+  },
+  "author": [
+    {
+      "given": "Dengyong",
+      "family": "Zhou"
+    },
+    {
+      "given": "Olivier",
+      "family": "Bousquet"
+    },
+    {
+      "given": "Thomas Navin",
+      "family": "Lal"
+    },
+    {
+      "given": "Jason",
+      "family": "Weston"
+    },
+    {
+      "given": "Bernhard",
+      "family": "Scholkopf"
+    }
+  ]
+},
+{
+  "type": "article",
+  "standard_citation": "raw:adamic",
+  "URL": "https://www.sciencedirect.com/science/article/pii/S0378873303000091",
+  "publisher": "Social Networks",
```

Comment:
Should put DOI in here https://doi.org/10.1016/S0378-8733(03)00009-1. Is manual reference metadata just to workaround the problem with the citation tag containing `(`? If so, can use shortDOI or tag.